source/core/NumericString.cpp

```diff
@@ -206,7 +206,7 @@ void ReadPercentFormatOptions(SubString *format, FormatOptions *pOptions)
 }
 //---------------------------------------------------------------------------------------------
 void GenerateFinalNumeric (const FormatOptions* , char* , Int32* , TempStackCString* , Boolean );
-void RefactorLabviewNumeric(const FormatOptions* , char* , Int32* , Int32 , Int32 );
+void RefactorLVNumeric(const FormatOptions* , char* , Int32* , Int32 , Int32, Boolean );
```

Comment:
Consider removing "LV" as well. As people start using Vireo for other purposes having references to LV will make less sense.